contracts/libraries/QuotasLogic.sol

```diff
@@ -32,7 +32,7 @@ library QuotasLogic {
         returns (uint128)
     {
         // `quoted` is `uint96`, and `cumulativeIndex / RAY` won't reach `2 ** 32` in reasonable time, so casting is safe
-        return quoted <= 1 ? 0 : uint128(uint256(quoted) * (cumulativeIndexNow - cumulativeIndexLU) / RAY); // U:[QL-2]
+        return uint128(uint256(quoted) * (cumulativeIndexNow - cumulativeIndexLU) / RAY); // U:[QL-2]
```

Comment:
Consider using the mulDiv function or preventing phantom overflow in some other way.